java/src/main/java/com/google/appengine/tools/pipeline/impl/model/JobRecord.java

```diff
@@ -410,7 +412,15 @@ public static boolean isExceptionHandlerSpecified(Job<?> jobInstance) {
     }
     return result;
   }
-
+  
+  // converts the settings array into a map
+  private void createSettingsMap(JobSetting[] settings) {
```

Comment:
I would prefer having this method as static and returns a map (to be assigned by the caller).